my/react-props-app/src/customButton.js

```diff
@@ -0,0 +1,3 @@
+export default function CustomButton({text,color}) {
```

Comment:
Usually we would make the file name match the component name. (So in this case the file name is slightly off -- it should start with uppercase.)